net-libs/libnma/libnma-1.10.4.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg vala
+
+DESCRIPTION="NetworkManager GUI library"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager"
+
+LICENSE="GPL-2+"
+SLOT="0"
+# pkcs11 default enabled as it's a small dep often already present by libnma users, and it was default enabled as IUSE=+gcr in nm-applet before
+IUSE="gtk-doc +introspection +pkcs11 vala"
+REQUIRED_USE="vala? ( introspection )"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+DEPEND="
+	>=gui-libs/gtk-4.0:4
+	app-text/iso-codes
+	net-misc/mobile-broadband-provider-info
+	>=dev-libs/glib-2.38:2
+	>=x11-libs/gtk+-3.12:3[introspection?]
+	>=net-misc/networkmanager-1.7[introspection?]
+	pkcs11? ( >=app-crypt/gcr-3.14:0=[gtk] )
```

Comment:
https://gitlab.gnome.org/GNOME/libnma/-/commit/2116cbd272ec2d9bb9e946ca48afd2c5e59d29cf appears to have changed the gcr dependency. Previously the `gcr_gtk4` meson option selected between gcr-4 (`app-crypt/gcr:4`) and gc3-3 (`app-crypt/gcr:0`). It looks to me like it automatically chooses whatever is available, with a preference for gcr-4.

I think we should just depend on `app-crypt/gcr:4`. Additionally, it says that the Gtk/UI portion of gcr isn't needed, so we shouldn't need the `[gtk]` USE-dep anymore.